rewrite-java/src/main/java/org/openrewrite/java/internal/VariableNameUtils.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.openrewrite.java.search;
+package org.openrewrite.java.internal;
```

Comment:
Curious why this is in `internal`?

Reply:
yep, exactly, I think we fully expect our recipe authors to need this. If we see them using the static method directly, it should probably be in `org.openrewrite.java`.

Reply:
That makes sense, it'll probably be used outside of utility methods based on the circumstance.
I moved the util into `org.openrewrite.java`